Makefile

```diff
@@ -53,6 +53,10 @@ build: check-version dep
 
 e2e: export MESG_PATH = $(PWD)/e2e.test/mesg
 e2e: clean-e2e
+	# setup - copy config to MESG_PATH
```

Comment:
what about a new command to compile the test service using the mesg-cli? It's not a really nice solution but it prevents to test against an old test

Reply:
Yes I can add the new target, but let's not do that in this PR. 

Btw, It won't prevent testing old service if someone forgets to run it. We can do this automatically (by depended targets) and get rid of `test-service.json` from repo, but in that case we have to prepare docker with  mesg-cli to be used in cirlceci.

Reply:
we could use `npx mesg-cli ....` that download mesg-cli if it not installed ;)